packages/components/src/select-control/index.tsx

```diff
@@ -39,7 +41,7 @@ function UnforwardedSelectControl(
 		label,
 		multiple = false,
 		onBlur = noop,
-		onChange = noop,
+		onChange,
```

Comment:
Breaking change?

Reply:
It shouldn't be, since I've added optional chaining when calling it in the file, which should have the same effect as defaulting to `noop`